eng/pipelines/templates/jobs/ci.yml

```diff
@@ -335,6 +335,14 @@ jobs:
           arguments: >
             -Directory sdk/${{ parameters.ServiceDirectory }}
 
+      - task: PowerShell@2
+        displayName: Verify Cadl Code Generation
+        inputs:
+          pwsh: true
+          filePath: $(Build.SourcesDirectory)/eng/pipelines/scripts/Cadl-Compare-CurrentToCodegeneration.ps1
+          arguments: >
+            -Directory sdk/${{ parameters.ServiceDirectory }}
```

Comment:
Curently I added the service directory parameter to make it consistent with the previous code verification with swagger/readme.md. But from my point of view, it might be more useful to not set argument so the script will regenerate SDK for all folders which has cadl-location.yaml under it. Please let me know your thoughts.